src/pages-sections/why-abu-dhabi/Section2.jsx

```diff
@@ -0,0 +1,23 @@
+import { Box, Button, Grid, styled, useTheme } from "@mui/material";
+
+
+const GridItemTwo = styled(Grid)(({ theme }) => ({
+    [theme.breakpoints.down("sm")]: {
+        display: "none",
+    },
+    marginTop: "4rem",
+    fontSize: "18px",
```

Comment:
It's better if we have a common place where we store all the font sizes we are going to use so that we don't use different fonts in different pages